scripts/painterroConfiguration.js

```diff
@@ -0,0 +1,85 @@
+var painterroConfiguration = {
```

Comment:
It looks like a script, but it fits more to the data package, I think.

Reply:
it may be. I will move it to data directory